### PR TITLE
allow ingesting non data cubes

### PR DESCRIPTION
**Pull request by TonioF**
Solves https://github.com/bcdev/service-eurodatacube/issues/70.

Checklist:

* [x] Add unit tests and/or doctests in docstrings
~~*Add docstrings and API docs for any new/modified user-facing classes and functions~~
~~*New/modified features documented in `docs/source/*`~~
* [x] Changes documented in `docs/CHANGES.md`
* [ ] AppVeyor and Travis CI passes
* [ ] Test coverage remains or increases (target 100%)
* [ ] Associated issues closed after merge

### CODE REVIEW

File: xcube/cli/_gen2/open.py [hunk 1]
```diff
@@ -41,16 +43,30 @@ def open_cubes(input_configs: Sequence[InputConfig],
...
+                        if TYPE_SPECIFIER_CUBE.is_satisfied_by(type_specifier):
+                            opener_ids = store.get_data_opener_ids(
+                                data_id=input_config.data_id, type_specifier=type_specifier)
+                            break
                     if not opener_ids:
-                        raise DataStoreError(f'Data store "{input_config.store_id}" does not support data cubes')
+                        for type_specifier in type_specifiers:
+                            if TYPE_SPECIFIER_DATASET.is_satisfied_by(type_specifier):
+                                opener_ids = store.get_data_opener_ids(
+                                    data_id=input_config.data_id, type_specifier=type_specifier)
```
Comment (forman) [inline xcube/cli/_gen2/open.py:64]:
```suggestion
                                opener_ids = store.get_data_opener_ids(data_id=input_config.data_id, 
                                                                       type_specifier=type_specifier)
```

File: xcube/cli/_gen2/open.py [hunk 2]
```diff
@@ -41,16 +43,30 @@ def open_cubes(input_configs: Sequence[InputConfig],
...
+                                normalisation_required = True
+                                break
+                    if not opener_ids:
+                        raise DataStoreError(f'Data store "{input_config.store_id}" '
+                                             f'does not support data sets')
```
Comment (forman) [inline xcube/cli/_gen2/open.py:69]:
```suggestion
                                             f'does not support datasets')
```

File: xcube/cli/_gen2/open.py [hunk 3]
```diff
@@ -41,16 +43,30 @@ def open_cubes(input_configs: Sequence[InputConfig],
...
                 store = store_instance.store
                 if opener_id is None:
-                    opener_ids = store.get_data_opener_ids(data_id=input_config.data_id,
-                                                           type_specifier=TYPE_SPECIFIER_CUBE)
+                    opener_ids = None
+                    type_specifiers = store.get_type_specifiers_for_data(input_config.data_id)
+                    for type_specifier in type_specifiers:
+                        if TYPE_SPECIFIER_CUBE.is_satisfied_by(type_specifier):
+                            opener_ids = store.get_data_opener_ids(
+                                data_id=input_config.data_id, type_specifier=type_specifier)
```
Comment (forman) [inline xcube/cli/_gen2/open.py:58]:
```suggestion
                            opener_ids = store.get_data_opener_ids(data_id=input_config.data_id, 
                                                                   type_specifier=type_specifier)
```

File: test/cli/_gen2/test_main.py [hunk 4]
```diff
@@ -5,17 +5,21 @@
...
+from xcube.core.store import DataStorePool
 from xcube.core.store.stores.memory import MemoryDataStore
 
 
 class MainTest(unittest.TestCase):
     REQUEST = dict(input_config=dict(store_id='memory',
                                      data_id='S2L2A'),
                    cube_config=dict(variable_names=['B01', 'B02', 'B03'],
-                                    crs='WGS84',
+                                    crs='epsg:4326',
```
Comment (forman) [inline test/cli/_gen2/test_main.py:22]: What is the problem with "WGS84"?

File: CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 ## Changes in 0.6.2 (in development)
 
+* xcube gen2 may ingest non-cube datasets
```
Comment (forman) [inline CHANGES.md:3]:
```suggestion
* CLI tool `xcube gen2` may now also ingest non-cube datasets.
```

**Review by forman — CHANGES_REQUESTED** [review]
Good! Mostly formatting issues.

File: test/cli/_gen2/test_main.py (hunk 4 above)
Comment (TonioF) [reply test/cli/_gen2/test_main.py:22]: That's the problem with my pyproj installation. I revert this.

**Pull request merged**